docs/_templates/index.html

```diff
@@ -0,0 +1,255 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="description" content="">
+    <meta name="author" content="">
+
+    <title>Memray: the endgame memory profiler</title>
+
+    <!-- Bootstrap core CSS -->
+    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,700" rel="stylesheet">
+
+    <!-- Custom styles for this template -->
+    <link href=" {{ pathto('_static/css/custom.css', 1) }}" rel="stylesheet">
+
+    <style>
+        .m-intro:before {
+            background-image: url("{{pathto('_static/images/circle_inner_opt.png', 1)}}");
+        }
+
+        .m-intro:after {
+            background-image: url("{{pathto('_static/images/circle_outer_opt.png', 1)}}");
+        }
+    </style>
+
+</head>
+
+<body>
+
+    <header>
+        <nav class="navbar navbar-expand-md fixed-top">
+            <div class="navbar-brand">
+                <img src=" {{ pathto('_static/images/logo.png', 1) }} " style="height: 2rem;" alt="Memray logo">
+            </div>
+
+            <div class="navbar-light bg-light">
+                <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavDropdown"
+                    aria-controls="navbarNavDropdown" aria-expanded="false" aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+            </div>
+
+            <div class="collapse navbar-collapse" id="navbarNavDropdown">
+                <ul class="navbar-nav mr-auto">
+                    <li class="nav-item active">
+                        <a class="nav-link" href="#">Home <span class="sr-only">(current)</span></a>
+                    </li>
```

Comment:
Why have this entry? It points to the page that we're currently on, and it can never point to any other page - why have it at all?